site/content/en/guides/producer/setters/_index.md

```diff
@@ -87,6 +87,57 @@ spec:
   replicas: 3 # {"$kpt-set":"replicas"}
 ```
 
+#### Targeting fields using the path
+
+The basic way to create a setter only matches fields based on the value. But
+in some situations there might be several fields with the same value, but not all
+of them should be targeted by the setter. In these situations, use the `--field`
+flag to provide either the name of the field, the full path to the field, or a
+partial (suffix) path to the field. Only fields that match the path will be
+targeted by the setter.
```

Comment:
I think what's implicit and must be said upfront is the AND logic here. Both Value AND path must match. In the example below also highlight this.

Reply:
Updated.